packages/reaction-core/common/methods/cart.js

```diff
@@ -4,8 +4,9 @@
 //
 if (Meteor.isClient) {
   Meteor.methods({
-    "cart/submitPayment": function (paymentMethod) {
+    "cart/submitPayment": function (paymentMethod, sessionId) {
```

Comment:
@aaronjudd, I think there is at least one way to not add `sessionId` to `cart/submitPayment` as arg.

We could untie `cartId` from `invoiceID`(PayPal or other). if we will do that, then we could avoid to remove cart on each order, but this could lead to more and more breaking changes. So this could be not the best scenario.